crates/rmf_site_editor/src/site/drawing.rs

```diff
@@ -249,7 +249,7 @@ pub fn update_drawing_rank(
                 if let Ok(segment) = measurements.get(*child) {
                     transforms
                         .get_mut(**segment)
-                        .map(|mut tf| tf.translation.z = z + DEFAULT_MEASUREMENT_OFFSET)
+                        .map(|mut tf| tf.translation.z = z + layers::ZLayer::get_offset())
```

Comment:
This looks a bit suspicious.
If I understand the intention of the original code correctly, it tries to offset a measurement to be above the drawing by a fixed value.
Can we just do that by adding `ZLayer::Measurement.to_z() - ZLayer::Drawing.to_z()`? Note that in your enum definition the distance between drawings and measurement is "2 offsets" since they are two enum variants away, this would make it only "one offset" that would actually cause Z fighting with floors.

I also lowkey wonder if this call was the only reason `get_offset()` is public and whether we should just make it private instead.